packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart/index.ts

```diff
@@ -0,0 +1,4 @@
+
+export {ForceFunctionContainerlessStartExecutor} from './ForceFunctionContainerlessStartExecutor';
+export {ForceFunctionContainerStartExecutor} from './ForceFunctionContainerStartExecutor';
+export { validateStartFunctionsUri } from './validateStartFunctionsUri';
```

Comment:
hum...I'm not sure why prettier didn't catch the space inconsistency in these exports.

Reply:
Don't think we have a rule for this. But for consistency, I see we use space in both exports and imports.

Reply:
hum...yeah I double checked and prettier should take care of this.  I'm not sure how it slipped by the first time.